Cargo.toml

```diff
@@ -16,6 +16,10 @@ log = "0.4"
 regex = "1.1"
 nix = "0.20.0"
 libc = "0.2"
+serde = { version = "1.0", features = ["derive"], optional = true }
 
 [dev-dependencies]
 libc = "0.2.76"
+
+[features]
+default = ["serde"]
```

Comment:
enabled by default? but you have said: "The feature is turned off by default" in the description.

Reply:
Fixed. Thanks. I removed the default feature `serde`.